pkg/kubelet/sysctl/runtime.go

```diff
@@ -35,15 +35,15 @@ const (
 // TODO: The admission logic in this file is runtime-dependent. It should be
 // changed to be generic and CRI-compatible.
 
-type runtimeAdmitHandler struct {
+type AdmitHandler struct {
 	result lifecycle.PodAdmitResult
 }
 
-var _ lifecycle.PodAdmitHandler = &runtimeAdmitHandler{}
+var _ lifecycle.PodAdmitHandler = &AdmitHandler{}
 
 // NewRuntimeAdmitHandler returns a sysctlRuntimeAdmitHandler which checks whether
 // the given runtime support sysctls.
-func NewRuntimeAdmitHandler(runtime container.Runtime) (*runtimeAdmitHandler, error) {
+func NewAdmitHandler(runtime container.Runtime) (*AdmitHandler, error) {
```

Comment:
Is there any reason not to return `lifecycle.PodAdmitHandler` directly? I don't think we'd need to export the `runtimeAdminHandler` type.

Reply:
Hmm, I believe that `runtimeAdmitHandler` has an `Admit` method defined on it - so if we were to get rid of the `runtimeAdmitHandler` type in favor of directly returning `lifecycle.PodAdmitHandler`, we would need to find a place for `runtimeAdmitHandler.Admit` to live... I guess it could just become a function that takes `lifecycle.PodAdmitHandler` as an argument?

Reply:
@yujuhong does we can update runtimeAdmitHandler into a function that takes ` lifecycle.PodAdmitHandler `  as an argument ?Do you have any better Suggestions?